components/Header.js

```diff
@@ -14,4 +14,13 @@ export const Title = styled.h1`
   right: 0;
   left: 0;
   top: 0;
+
+  img {
+    position: fixed;
+    left: 0;
+    top: 0;
+    mix-blend-mode: normal;
```

Comment:
This doesn't do anything ;)

Reply:
Another sidenote: I'm not super convinced that using a `h1` for your header is a great idea. The `h1` in this instance does declare the most valuable information of your app, which would be that it's an RPG Sheet-Creator - but: users would usually excpect an `h1` to be part of the app's content, rather than a header.

If your header is this:
`<div><img></img><span>RPG Sheet-Creator</span></div>`
you can use this as a welcoming message:
`<h1>Welcome to K. Ohlmann's RPG Sheet-Creator</h1>`

Google's web-crawlers will more likely grip onto that `h1` (and people using screenreaders won't be confused), and you can include other pertinent information, etc.